### PR TITLE
Feat: workflow crud and fix dataflow

### PR DESCRIPTION
**Pull request by wangbow**
Feat: workflow crud and fix dataflow

### CODE REVIEW

File: src/pages/ApplicationWorkflow/workflow-component/index.tsx [hunk 1]
```diff
@@ -166,7 +193,7 @@ class WorkFlowComponent extends Component<Props, State> {
                   this.saveWorkflow();
                 }}
               >
-                保存
+                <Translation>Save to Server</Translation>
```
Comment (barnettZQG) [inline src/pages/ApplicationWorkflow/workflow-component/index.tsx:196]:
```suggestion
                <Translation>Save</Translation>
```

File: src/pages/ApplicationWorkflow/workflow-component/index.tsx [hunk 2]
```diff
@@ -138,7 +165,7 @@ class WorkFlowComponent extends Component<Props, State> {
                       })}
                     />
                   </Form.Item>
-                  <Form.Item label="工作流别名">
+                  <Form.Item label={<Translation>Workflow alias</Translation>}>
```
Comment (barnettZQG) [inline src/pages/ApplicationWorkflow/workflow-component/index.tsx:168]:
```suggestion
                  <Form.Item label={<Translation>Workflow Alias</Translation>}>
```

File: src/pages/ApplicationWorkflow/workflow-component/index.tsx (hunk 1 above)
Comment (wangbow) [reply src/pages/ApplicationWorkflow/workflow-component/index.tsx]: done

File: src/pages/ApplicationWorkflow/workflow-component/index.tsx (hunk 2 above)
Comment (wangbow) [reply src/pages/ApplicationWorkflow/workflow-component/index.tsx]: done

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**